rosdep/base.yaml

```diff
@@ -5558,7 +5558,7 @@ opencl-headers:
 opende:
   arch: [ode]
   debian: [libode-dev]
-  fedora: [ode]
+  fedora: [ode-devel]
```

Comment:
I guess it's a Fedora convention to append the `-devel` to the package name? I don't see a package listed with the name `ode-devel`.

Reply:
It works similarly to how it does on Ubuntu. Typically, the `-devel` entity is created as a subpackage.

https://src.fedoraproject.org will show you source packages, and doesn't show the subpackages created by those source packages. If you click one of the specific package versions listed in the updates, you'll link to the build system, Koji. That will show the subpackages that were created by that source package when it was built: https://koji.fedoraproject.org/koji/buildinfo?buildID=1684361

Unfortunately, Fedora doesn't provide a good web frontend similar to https://packages.ubuntu.com that also contains subpackages.

The role of the `-devel` package on Fedora is the same as the `-dev` packages on Ubuntu: you typically get headers, CMake stuff, pkg-configs, and unversioned SO library links.

Reply:
Great, thanks for the detailed breakdown :bow: